scripts/python/generate_dbt_yaml.py

```diff
@@ -176,7 +177,7 @@ def row_handler(row: Tag) -> OmopDocumentationContainer:
 
     # Remove dangling whitespace and newlines from parsed HTML
     cells_stripped: dict[str, str] = {
-        k: v.replace("\n", "").strip() for k, v in cells_raw.items()
+        k: v.replace("\n", "").strip().replace("“", "").replace("”", "") for k, v in cells_raw.items()
```

Comment:
the quotes on the offset column in the CDM spec were causing a failure in the column presence dbt test